### PR TITLE
Corrected several issues highlighted by pylint

### LINKED ISSUE #43: `pylint` gives gives warnings
I checked out the `HEAD` (15e8c693) of `main` and ran `pylint` as suggested in the `README`:
```
$ pylint --rcfile=scripts/.pylintrc scripts
************* Module make
scripts/make.py:607:4: C0116: Missing function or method docstring (missing-function-docstring)
************* Module repos
scripts/repos.py:162:0: C0330: Wrong hanging indentation (remove 2 spaces).
              repo, branch_str, status['SHA1'], status['Dirty']))
            | ^ (bad-continuation)
scripts/repos.py:26:0: E0401: Unable to import 'git' (import-error)
************* Module build
scripts/build.py:60:47: C0326: Exactly one space required after comma
    parser.add_argument('--build-dir', type=str,  metavar='PATH',
                                               ^ (bad-whitespace)
scripts/build.py:146:33: C0326: Exactly one space required after comma
    parser.add_argument('actions',  nargs=argparse.REMAINDER,
                                 ^ (bad-whitespace)
************* Module config
scripts/config.py:162:42: C0326: Exactly one space required after comma
        self.name = '{}_{}_{}'.format(arch,  float_abi.value, name_suffix)
                                          ^ (bad-whitespace)
scripts/config.py:227:0: C0330: Wrong continued indentation (remove 3 spaces).
                                  action != Action.TEST)
                               |  ^ (bad-continuation)
scripts/config.py:226:34: R1714: Consider merging these comparisons with "in" to 'action not in (Action.ALL, Action.TEST)' (consider-using-in)
------------------------------------------------------------------
Your code has been rated at 9.89/10 (previous run: 9.89/10, +0.00)
```

My version of `pylint` is as follows:

```
$ pylint --version
pylint 2.4.2
astroid 2.3.1
Python 3.8.5 (default, Jan 27 2021, 15:41:15)
[GCC 9.3.0]
```

### PR DESCRIPTION
**Pull request by danhag01**
closes #43

### CODE REVIEW

File: scripts/repos.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import sys
 from typing import Dict, List, Any
 
+# pylint: disable=import-error
```
Comment (miyuki) [inline scripts/repos.py:26]: This does not look right. Are you running pylint from a virtual environment with installed dependencies?

**Review by miyuki — COMMENTED** [review]
Please rebase your branch on top of main and get rid of the merge commit.

File: scripts/repos.py (hunk 1 above)
Comment (danhag01) [reply scripts/repos.py:26]: Yes, I'm running in a virtual environment but it appears to be misconfigured. I'll remove this line.

File: scripts/make.py [hunk 2]
```diff
@@ -577,10 +577,10 @@ def _check_output(output_type: str, smoketest_dir: os.DirEntry,
                 expected_output_path = os.path.join(smoketest_path,
                                                     output_type)
                 try:
-                    expected_output = open(expected_output_path)
-                    expected_output_str = [
-                        s.rstrip('\n') for s in expected_output.readlines()
-                    ]
+                    with open(expected_output_path) as expected_output:
```
Comment (danhag01) [inline scripts/make.py:580]: `pylint` had the following:

`scripts/make.py:580:20: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)`

**Comment by miyuki** [comment]
The patch looks fine. Please rebase it on top of main and squash into a single commit.

**Review by miyuki — APPROVED** [review]

**Pull request merged**